examples/asr/asr_chunked_inference/rnnt/speech_to_text_buffered_infer_rnnt.py

```diff
@@ -214,7 +217,7 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
 
     # Change Decoding Config
     with open_dict(cfg.decoding):
-        if cfg.stateful_decoding:
+        if cfg.stateful_decoding or cfg.merge_algo == 'tdt':
```

Comment:
Add this info to the doc string above and also add to script usage at the top of this script.

Reply:
Done.